noir/noir-repo/test_programs/execution_success/databus_two_calldata_simple/src/main.nr

```diff
@@ -0,0 +1,4 @@
+fn main(mut idx: u32, y: call_data(0) u32, z: call_data(1) [u32; 4]) -> return_data u32 {
```

Comment:
As expected, this program results in 3 databus block constraints, two for calldata and one for return_data.